README.md

```diff
@@ -84,7 +84,12 @@ Tests are an important part of the development and evolution of this project,
 and new contributions are expected to include tests for any functionality
 change.  To run the tests, pass the `test` verb to the `bootstrap` script:
 
-    ./Utilities/bootstrap --build-tests test
```

Comment:
By default `Utilities/bootstrap` uses `swiftc` and other tools from the Toolchains specified in the `PATH` and it works for me with latest toolchain on MacOS at least. 
On linux it could fail because of this https://github.com/apple/swift-package-manager/pull/158

I think it wold be useful to have both options described in readme, because often you need to use swiftc from `maser`.